modules/deployment/src/test/java/org/mule/runtime/module/deployment/test/internal/ApplicationPolicyDeploymentTestCase.java

```diff
@@ -373,6 +381,51 @@ public void reorderApplicationPolicy() throws Exception {
     assertThat(policyParametrization, startsWith(BAR_POLICY_NAME + FOO_POLICY_NAME));
   }
 
+  @Test
+  @Story(POLICY_SILENT_DEPLOY)
+  public void silentApplicationPolicyDoesNotLogSplash() throws Exception {
```

Comment:
```suggestion
  @Story(POLICY_SILENT_DEPLOY)
  public void silentApplicationPolicyLogsWithDebugLevel() throws Exception {
```
wouldn't this capture the expectation of the test a little better?